### PR TITLE
built the cart page

### PR DESCRIPTION
**Pull request by devAmlan**
Built the cart page of e-commerce app.

Added features like
1. Add to cart
2. Remove from cart
3. Increase and decrease the quantity.

Preview ->
mlanstore-cartpage

Please review my PR .

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -4,31 +4,36 @@ import Navbar from './components/navbar/Navbar';
...
+                  <Route path="/signup" element={<Signup />} />
+                  <Route path='/login' element={<Login />} />
+                  <Route path='/cart' element={<Cart />} />
+                </Routes>
+                <Bottombar />
+              </FilterProvider>
+            </CartProvider>
+          </ProductProvider>
+        </AuthProvider>
+      </BrowserRouter>
```
Comment (iRohitGaur) [inline frontend/src/App.js:36]: suggestion: you can put the providers in `index.js`

**Review by iRohitGaur — APPROVED** [review]
Nice work!

Good to merge 👍🏻

File: frontend/src/components/product/Product.js [hunk 2]
```diff
@@ -1,10 +1,17 @@
...
-function Product({ productname, productimage, productprice, originalprice, producttag, productrating }) {
+import { AiFillHeart, AiOutlineHeart, AiFillStar } from "react-icons/ai"
+import { CartContext } from "../../context/CartContext"
+import { ToastContainer } from "react-toast"
+function Product({ productid,productname, productimage, productprice, originalprice, producttag, productrating }) {
+
   const [addwishlist, setAddwishlist] = useState(false)
+  const { addTocart } = useContext(CartContext)
+  const product = {productid,productname, productimage, productprice, originalprice, producttag, productrating}
+
```
Comment (bhushannaikaroskar) [inline frontend/src/components/product/Product.js:11]: Please use camel case properly to name variables you can use productId, productName, etc.

**Review by bhushannaikaroskar — CHANGES_REQUESTED** [review]
Hey Amlan reviewed your code for Cart page UI seems good. I have given some suggestions for variable naming. Please do implement it. I will approve once you are done with these changes

File: frontend/src/components/product/Product.js (hunk 2 above)
Comment (devAmlan) [reply frontend/src/components/product/Product.js:11]: Thanks for your suggestion @bhushannaikaroskar , I will change the names into camel case.

File: frontend/src/App.js (hunk 1 above)
Comment (devAmlan) [reply frontend/src/App.js:36]: Thanks for the suggestion @iRohitGaur ,I will shift those providers to index.js.

**Pull request merged**